### ISSUE TITLE
TableBatchAction iconDescription should only be required when there are no children and should default to ''

### ISSUE THREAD

**Issue by justindm234** [carbon-design-system/carbon#3809]
If an iconDescription is defined, it is read by JAWS in addition to the button's visible label. So an iconDescription should only be provided when there is no visible label. 

![image](https://user-images.githubusercontent.com/15637876/63607806-5fd96500-c5a1-11e9-96b8-acb91b7a68f1.png)

Additionally, callers must pass iconDescription='' to override the bad default value of 'Add' (the default value should be '')

![image](https://user-images.githubusercontent.com/15637876/63607888-92835d80-c5a1-11e9-84ab-7b18f9b017d4.png)

## Environment

Firefox ESR 60.8.0esr
JAWS Version 2019.1906.10 ILM

## Detailed description

carbon-components@10.5.1
carbon-components-react@7.5.1

## Steps to reproduce the issue

https://codesandbox.io/s/tablebatchaction-icondescription-provided-7jk8o
https://codesandbox.io/s/tablebatchaction-icondescription-not-provided-v11z8
